data_transfer/devices/dreem.py

```diff
@@ -224,6 +225,12 @@ def download_file(self, mongo_id: str) -> None:
             self.session, record.download_folder(), record.manufacturer_ref
         )
 
+        # Useful metadata for performing pre-processing.
+        downloaded_file = Path(
+            record.download_folder() / f"{record.manufacturer_ref}.h5"
+        )
+        record.meta = dict(filesize=downloaded_file.stat().st_size)
```

Comment:
This would override existing metadata I believe - perhaps directly set the field? i.e.:
```python
record.meta.filesize = downloaded_file.stat().st_size

```